open-craft/learner-profile-dashboard
# Address post-delivery feedback

cf. [OC-4615](https://tasks.opencraft.com/browse/OC-4615)

This PR adds logic that allows the LPD to localize submission data to the learner's timezone when displaying it. It also makes sure that the LPD will load for users accessing a course via Safari.

**Test instructions**



Using Chrome/Firefox:

1. On [LPD stage](https://hgse-stage.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/jump_to/block-v1:edX+DemoX+Demo_Course+type@vertical+block@45c7cedb4bfe46f4a68c78787151cfb5), answer some questions and click "Submit your answers". Observe that the time of the most recent submission (displayed next to the "Submit your answers" button) changes to the current time from your timezone.



Using Safari:

1. Clear cache in Safari ("Develop" > "Empty Caches").

1. Launch new private window.

1. [Access LPD](https://studio-hgse-stage.opencraft.hosting/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@45c7cedb4bfe46f4a68c78787151cfb5) in Studio of HGSE stage. Observe that the LPD does *not* display the Django admin login page, but rather a page that explains why actual content can't be loaded directly.

1. Follow instructions on that page (click button and refresh page once temporary window closes). Observe that the LPD now displays appropriate content.

1. Open ["View Live Version"](https://hgse-stage.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/jump_to/block-v1:edX+DemoX+Demo_Course+type@vertical+block@45c7cedb4bfe46f4a68c78787151cfb5) in a new tab and log in, then repeat the process described in the previous two steps. 

**Reviewers**
